browser/src/control/Control.ContextMenu.js

```diff
@@ -214,6 +215,14 @@ L.Control.ContextMenu = L.Control.extend({
 			$('.context-menu-root').focus();
 			this.hasContextMenu = true;
 		}
+		const contextMenuLayer = document.getElementById('context-menu-layer');
```

Comment:
Please try to avoid using direct usage of generated DOM elements by some libraries.
I think context menu has own mechanics to add "on click outside" handler - when we create it

Reply:
[ browser/src/control/Control.ContextMenu.js ](https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.ContextMenu.js#L207)

Reply:
did you check that? hide thing?

Reply:
I missed that. Thank you Szymon. I updated the patch